datarobot_batch_scoring/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.8.8'
+__version__ = '1.9.0'
```

Comment:
I figure this would be a good place to do a minor bump because these changes are more likely to introduce instability. We can always say "go back to 1.8.x "  if there are issues.